app/views/ReportingFrequencyView.scala.html

```diff
@@ -431,11 +488,17 @@ <h2 class="govuk-summary-card__title" id="manage-reporting-obligations-card-head
         @manageReportingFrequency
     }
 
-    @if(!viewModel.displayCeasedBusinessWarning) {
+    @if(!viewModel.displayCeasedBusinessWarning && !viewModel.showYourReportingObligations) {
         @differentObligations
     }
 
+    @if(viewModel.showYourReportingObligations) {
+        @yourReportingObligations
+    }
```

Comment:
Could we rename YourReportingObligations to be a little more descriptive? Its a little hard to tell that it's for specifically the exempt version.